apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue

```diff
@@ -150,30 +221,51 @@ const state = reactive({
         });
         return results;
     }),
+    prevTotalFailureCount: computed(() => random(0, 200)), // TODO: real data
+    totalFailureCount: computed(() => sum(state.data?.map((d) => d.fail_finding_count))),
```

Comment:
issue: We need to separate the case when the data is zero and the case when the data is empty.